internal/service/encryptionatrest/resource_encryption_at_rest.go

```diff
@@ -173,6 +176,13 @@ func (r *encryptionAtRestRS) Schema(ctx context.Context, req resource.SchemaRequ
 							Optional:  true,
 							Sensitive: true,
 						},
+						"require_private_networking": schema.BoolAttribute{
+							Optional: true,
+							Computed: true,
+							PlanModifiers: []planmodifier.Bool{
+								boolplanmodifier.UseStateForUnknown(),
+							},
```

Comment:
curious if this was needed to avoid non empty plans, or simply to remove noise in plan outputs.

Reply:
to remove noise in plan outputs